### PR TITLE
add in distributed for build catalog example

### PR DESCRIPTION
**Pull request by mgrover1**
## Change Summary

Try to fix the building docs error by adding in distributed to catalog building notebook.

## Checklist

- [x] Unit tests for the changes exist
- [x] Tests pass on CI
- [x] Documentation reflects the changes where applicable

### CODE REVIEW

File: docs/source/how-to/build-a-catalog-from-timeseries-files.md [hunk 1]
```diff
@@ -58,11 +58,11 @@ The only parts of ecgtools we need are the `Builder` object and the `parse_cesm_
 import pathlib
 
 import dask
+import distributed
```
Comment (andersy005) [inline docs/source/how-to/build-a-catalog-from-timeseries-files.md:61]: don't we need to instantiate a local cluster for this to have an effect?

Comment (mgrover1) [reply docs/source/how-to/build-a-catalog-from-timeseries-files.md:61]: Right - that needs to be removed too

Comment (mgrover1) [inline docs/source/how-to/build-a-catalog-from-timeseries-files.md:61]:
```suggestion
```

**Review by andersy005 — APPROVED** [review]
Looks great :). thank you, @mgrover1!

**Pull request merged**